frontend/src/metabase/static-viz/components/Funnel/Funnel.tsx

```diff
@@ -0,0 +1,180 @@
+        {steps.map((step, index) => {
+          const isFirst = index === 0;
+          const isLast = index === steps.length - 1;
+          const nextStep = steps[index + 1];
+
+          const points = isLast
+            ? null
+            : calculateFunnelPolygonPoints(step, nextStep, marginTop);
+          const opacity = calculateStepOpacity(index, steps.length);
+
+          const stepName = truncateText(step.step, maxStepTextWidth);
+          const measure = truncateText(
+            step.measure.toString(),
+            maxStepTextWidth,
+          );
+          const percent = truncateText(
+            formatPercent(step.percent),
+            maxStepTextWidth,
+            percentCharWidth,
+          );
+
+          return (
+            <>
+              {points && (
+                <Polygon
+                  key={index}
+                  fill={palette.brand}
+                  points={points}
+                  opacity={opacity}
+                />
+              )}
+              <Line
+                x1={step.left}
+                y1={0}
+                x2={step.left}
+                y2={layout.height}
+                stroke={palette.border}
+              />
+
+              <Group left={step.left - layout.stepTextOffset}>
+                <Text
+                  textAnchor="end"
+                  y={layout.stepTopOffset}
+                  fontSize={layout.stepFontSize}
+                  fill={palette.textMedium}
+                >
+                  {stepName}
+                </Text>
+
+                {isFirst && (
+                  <>
+                    <Text
+                      textAnchor="end"
+                      fontWeight={700}
+                      y={firstMeasureTop}
+                      fontSize={layout.initialMeasureFontSize}
+                      fill="black"
+                    >
+                      {step.measure}
+                    </Text>
+
+                    <Text
+                      textAnchor="end"
+                      fill={palette.textMedium}
+                      y={stepLabelTop}
+                      fontSize={layout.nameFontSize}
+                    >
+                      {settings.step.name}
+                    </Text>
+                  </>
+                )}
+
+                {!isFirst && (
+                  <>
+                    <Text
+                      textAnchor="end"
+                      fill={palette.textMedium}
+                      y={layout.height - layout.percentBottomOffset}
+                      fontSize={layout.percentFontSize}
+                    >
+                      {percent}
+                    </Text>
+
+                    <Text
+                      textAnchor="end"
+                      fill={palette.textMedium}
+                      y={layout.height - layout.measureBottomOffset}
+                      fontSize={layout.measureFontSize}
+                    >
+                      {measure}
```

Comment:
`Measure` column can have formatting settings, this should be taken into account. 

<img width="1679" alt="Screen Shot 2021-11-26 at 8 37 10 pm" src="https://user-images.githubusercontent.com/8542534/143615924-71e77faf-c648-44d2-a3bd-c81b0f8a6f2d.png">